edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Http/IRegistryApiClient.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Http
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    public interface IRegistryApiClient
+    {
+        public Task<HttpResponseMessage> PutModuleOnBehalfOfAsync(string actorEdgeDeviceId, CreateOrUpdateModuleOnBehalfOfData requestData);
```

Comment:
Why do these need to be declared public?

Reply:
will clean it up.